components/link/link.scss

```diff
@@ -14,12 +14,17 @@
 		color: $d2l-color-celestine-minus-1;
 		text-decoration: underline;
 	}
-	&:focus,
-	&.d2l-link-focus {
+	&:focus-visible {
 		outline: 2px solid $d2l-color-celestine;
 		outline-offset: 1px;
 		text-decoration: underline;
 	}
+	.d2l-focus-visible-not-supported &:focus {
```

Comment:
I think this can just be combined with the above rule? I'd want to inspect the CSS that this outputs though since remember at least old-Safari chokes on everything in a list of rules if it contains `:focus-visible`.

Reply:
I can look into that. Yeah, that was my worry... that Safari would choke.

Reply:
If I combine them, then this is what it produces:
```css
.d2l-test-link:focus-visible,
.d2l-focus-visible-not-supported .d2l-test-link:focus {
  outline: 2px solid #006fbf;
  outline-offset: 1px;
  text-decoration: underline;
}
```
Which from my understanding, we wouldn't want.

Reply:
Correct. Ah well, thanks for trying!